src/Common/src/System/IO/StringBuilderCache.cs

```diff
@@ -66,6 +66,13 @@ public static StringBuilder Acquire(int capacity = DEFAULT_CAPACITY)
             return new StringBuilder(capacity);
         }
 
+        public static StringBuilder Acquire(string value)
```

Comment:
I wonder if it is worth turning some of these APIs into extension methods on StringBuilder now. Originally when this code was written extension methods didn't exist.